cocotb/scheduler.py

```diff
@@ -749,13 +749,13 @@ async def wrapper():
         return wrapper()
 
     @staticmethod
-    def create_task(coroutine: Any) -> RunningTask:
+    def create_task(coroutine: Any) -> Task:
         """Checks to see if the given object is a schedulable coroutine object and if so, returns it"""
```

Comment:
```suggestion
        """Check to see if the given object is a schedulable coroutine object and if so, return it."""
```
While we are here...